components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -826,7 +828,10 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                         sshSubcycleCurWithTides(iCell) = &
                                  sshSubcycleCur(iCell) - &
                               tidalPotentialEta(iCell) - &
-                              config_self_attraction_and_loading_beta* &
+                              !!!!!!!!!!!!!!!! note config flag here is not set
+                              !correctly in init for tidal !!!!!!!!!
+                           ssh_sal_on * ssh_sal(iCell) - &
+                           (1 - ssh_sal_on) * config_self_attraction_and_loading_beta* &
```

Comment:
@philipwjones is it OK to use a real-valued config flag within a cell loop, or is it better to copy that to a local module variable on init?

Reply:
It's ok, but I prefer a local module variable for several reasons. (1) you can check on init for valid values, (2) you can make a shorter variable name and (3) the config values are currently pointers which on occasion can cause problems. We will be fixing (3) soon.